### PR TITLE
M.C.vanderKooij.Picto-Selector version 1.8.26

### PR DESCRIPTION
**Pull request by CasperNaudts**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/59515)

### CODE REVIEW

File: manifests/m/M/C/vanderKooij/Picto-Selector/1.8.26/M.C.vanderKooij.Picto-Selector.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: M.C.vanderKooij.Picto-Selector
```
Comment (Trenly) [inline manifests/m/M/C/vanderKooij/Picto-Selector/1.8.26/M.C.vanderKooij.Picto-Selector.installer.yaml:4]:
```suggestion
PackageIdentifier: MCvanderKooij.PictoSelector
```
Having the extra periods in the Identifier causes many sub-folders that aren't necessary

File: manifests/m/M/C/vanderKooij/Picto-Selector/1.8.26/M.C.vanderKooij.Picto-Selector.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: M.C.vanderKooij.Picto-Selector
```
Comment (Trenly) [inline manifests/m/M/C/vanderKooij/Picto-Selector/1.8.26/M.C.vanderKooij.Picto-Selector.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: MCvanderKooij.PictoSelector
```

File: manifests/m/M/C/vanderKooij/Picto-Selector/1.8.26/M.C.vanderKooij.Picto-Selector.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: M.C.vanderKooij.Picto-Selector
```
Comment (Trenly) [inline manifests/m/M/C/vanderKooij/Picto-Selector/1.8.26/M.C.vanderKooij.Picto-Selector.yaml:4]:
```suggestion
PackageIdentifier: MCvanderKooij.PictoSelector
```

File: manifests/m/M/C/vanderKooij/Picto-Selector/1.8.26/M.C.vanderKooij.Picto-Selector.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: M.C.vanderKooij.Picto-Selector
+PackageVersion: 1.8.26
+PackageLocale: en-US
+Publisher: M.C. van der Kooij
+PackageName: Picto-Selector
+License: Freeware
+ShortDescription: Picto-Selector is a free tool for creating visual schedules. It is used by many teachers and parents. But there are also reports of people using it in daycare for elderly people.
```
Comment (Trenly) [inline manifests/m/M/C/vanderKooij/Picto-Selector/1.8.26/M.C.vanderKooij.Picto-Selector.locale.en-US.yaml:10]:
```suggestion
ShortDescription: Picto-Selector is a free tool for creating visual schedules.
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**